pkg/proto/pbgo/core/api.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
-// 	protoc        v3.21.5
+// 	protoc-gen-go v1.30.0
```

Comment:
This is interesting.... Not sure why the compiler version is lower. The `inv -e install-tools` was supposed to have kept us aligned; there might be a bug or something in that invoke task. 

Can you confirm you called `inv -e install-tools` prior to generating the new definitions? If you did, then there's nothing for you to do, I can take a look and submit a follow-up PR if necessary.

Reply:
Yes I did call this at least once before regenerating the protobuf. That is curious...